public/app/app-init.js

```
@@ -1,644 +1,414 @@
-  // Attempt 1: elements with data-param attribute
-  document.querySelectorAll('input[type="range"][data-param]').forEach(el => {
-    const param = el.getAttribute('data-param');
-    if (!SLIDER_MAP[param]) return;
-    sliderParams[param] = parseFloat(el.value);
-    el.addEventListener('input', () => {
-      sliderParams[param] = parseFloat(el.value);
-      sendParamsToWorklet();
-      updateSliderValueDisplay(el, param);
-    });
-  });
+  App.mlWorker = new Worker('./ml-worker.js', { type: 'module' });
```

ESLint

[error] 'Worker' is not defined.

(no-undef)